react/components/SKUSelector/components/SelectorItem.js

```diff
@@ -22,14 +24,14 @@ export default class SelectorItem extends PureComponent {
     return (
       <div
         className={classNames(
-          'vtex-sku-selector__item relative di pointer flex items-center',
-          isImage && 'vtex-sku-selector__item-image'
+          `${styles.skuSelectorItem} relative di pointer flex items-center`,
+          isImage && `${styles.skuSelectorItemImage}`
```

Comment:
Here too

Reply:
In this case you should actually do something like this
```javascript
className={classNames(
          `${styles.skuSelectorItem} relative di pointer flex items-center`,
           {
             [styles.skuSelectorItemImage]: isImage
           })
```